site/_shortcodes/LanguageList.js

```diff
@@ -0,0 +1,53 @@
+const {findByUrl} = require('../_data/lib/find');
+const {getLocalizedPaths, stripLocale} = require('../_filters/urls');
+const path = require('path');
+
+/**
+ * A map of supported language codes to their full names.
+ * @const
+ */
+const languageNames = {
+  en: 'English',
+  es: 'Español',
+  ko: '한국어',
+  zh: '中文',
+  ru: 'Pусский',
+  pt: 'Português',
+  ja: '日本語',
+};
+
+/**
+ * Outputs a list of <link hreflang=""> tags with alternate language versions
```

Comment:
I think you mean `<a href="..."` (did you copy this comment from the head stuff?

Reply:
🙈🙈🙈